lldb/test/API/functionalities/process_save_core/TestProcessSaveCore.py

```diff
@@ -21,6 +21,7 @@ def test_cannot_save_core_unless_process_stopped(self):
         target = self.dbg.CreateTarget(exe)
         process = target.LaunchSimple(None, None, self.get_process_working_directory())
         self.assertNotEqual(process.GetState(), lldb.eStateStopped)
+        options = SBCoreDumpOptions(core)
```

Comment:
This won't work anymore unless you make the changes mentioned above... The API has changed and `SBCoreDumpOptions:: SBCoreDumpOptions()` doesn't take an argument anymore